src/functionHandlers/notarisePdt/validateInputs/mock_image.json

```diff
@@ -0,0 +1,3 @@
+}
```

Comment:
Anyone has any ideas on how to programmatically create a base64 image string (of a specified size)? So we can remove this file.